BLSQ/iaso
# Wc2-106: small adjustments to the get storage blacklisted endpoint

The response to GET /api/mobile/storage/blacklisted had a few differences to what was specified in https://bluesquare.atlassian.net/browse/WC2-65, in particular:

.- `updated_at` and `created_at` fields were returned at the root (device) and shouldn’t be.
- the `status` dict/object should be renamed `storage_status`
- `storage_status.updated_at` returned an ISO8601 string rather than a timestamp
- details about the orgunit and entity linked to the storage should not be returned

## Self proof reading checklist


## How to test

- Make sure you have at least one device in the database with the `BLACKLISTED` status
- Go to `/api/mobile/storage/blacklisted/`
- Make sure the response conforms to what is described at https://bluesquare.atlassian.net/browse/WC2-65
- (after the mobile app has been updated): make sure the interactions to detect the blacklisted devices still work

Explain how to test your PR.
If a specific config is required explain it here (dataset, account, profile, ...)

@bmonjoie: I think the backend now complies fully with the specs, can you do the adjustments to the mobile app and make sure all works as expected?
